### PR TITLE
Adding spec file for log4cpp and log4cpp-devel rpms

### PR DESCRIPTION
**Pull request by bkwzwz**
Creating a spec file to build log4cpp and log4cpp-devel rpms.  These 2 rpms will install liblog4cpp.so, liblog4cpp.a etc and the header files for Log4cpp library.

### CODE REVIEW

File: SPECS/log4cpp/log4cpp.spec [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+%files devel
+%defattr(-,root,root)
+%_prefix/include/*
+%_prefix/bin/log4cpp-config
+%_prefix/lib/lib*.so
+%_prefix/lib/*.*a
+%_prefix/lib/pkgconfig/log4cpp.pc
+%_prefix/share/aclocal/*.m4
+
+
```
Comment (ppadmavilasom) [inline SPECS/log4cpp/log4cpp.spec]: Please add changelog

Comment (bkwzwz) [reply SPECS/log4cpp/log4cpp.spec]: Done

**Comment by ppadmavilasom** [comment]
please follow existing specs for %{_includedir}, %{_bindir} etc as applicable. 
please also change this pull request to 1.0 branch instead of dev

**Comment by bkwzwz** [comment]
I will redo another PR since my source code was based on dev branch

**Comment by ppadmavilasom** [comment]
this is okay. @suezzelur pulled this to internal gerrit. we will address the outstanding issues in gerrit

**Comment by bkwzwz** [comment]
Do you still want me to create a new PR to merge to 1.0?

**Comment by ppadmavilasom** [comment]
No. This is okay. Thanks for the PR.

**Pull request closed without merging**